add generic guidelines #17 (#22)
MadhavBahlMD/my-open-profile: CONTRIBUTING.md

> ### Contributing

Reviewer: ```suggestion
# Contribution guidelines
```

- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.

> ### We Develop with Github

Reviewer: Not really necessary to have this section since the repository is hosted on GitHub anyways.

Author: Ok let me remove this

We use github to host code, to track issues and feature requests, as well as accept pull requests.

> ### We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
> - Pull requests are the best way to propose changes to the codebase (we use Github Flow). We actively welcome your pull requests:
> 

Reviewer: ```suggestion
## Workflow

Our project uses the [GitHub Flow](https://guides.github.com/introduction/flow/index.html)

The steps to make your first contribution to this project are as follows :

```

1. Fork the repo and create your branch from master.
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes.
5. Make sure your code lints.
6. Issue that pull request!

> ### Code of Conduct

Reviewer: Can be added as a separate Markdown document as [in the Atom repository for example](https://github.com/atom/atom/blob/master/CODE_OF_CONDUCT.md). If we wish to stick to a limited version, then we can add a Contributor Covenant Badge to the README.

Author: Right.

> ### Code of Conduct

Reviewer: ```suggestion
## Code of Conduct
```

> #### Our Pledge

Reviewer: ```suggestion
### Our Pledge
```

In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.

> #### Our Standards

Reviewer: ```suggestion
### Our Standards
```

# Contribution guidelines

- When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.

> 

Reviewer 2: ```suggestion
```

No need for extra blank line

## Workflow

Our project uses the [GitHub Flow](https://guides.github.com/introduction/flow/index.html)

The steps to make your first contribution to this project are as follows :

1. Fork the repo and create your branch from master.
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes.
5. Make sure your code lints.
6. Issue that pull request!

> ## Code of Conduct

Reviewer 2: ```suggestion
## Code of Conduct

```
There should be a blank line between 2 headings

### Our Pledge

> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.

Reviewer 2: ```suggestion

In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

### Our Standards

> Examples of behavior that contributes to creating a positive environment include:

Reviewer 2: ```suggestion

Examples of behavior that contributes to creating a positive environment include:
```

- Using welcoming and inclusive language
- Being respectful of differing viewpoints and experiences
- Gracefully accepting constructive criticism
- Focusing on what is best for the community
Showing empathy towards other community members

Examples of unacceptable behavior by participants include:

> - The use of sexualized language or imagery and unwelcome sexual attention or advances

Reviewer 2: ```suggestion

- The use of sexualized language or imagery and unwelcome sexual attention or advances
```
